tests/integration_tests/family/test_update.py

```diff
@@ -282,6 +282,51 @@ def test_update_family_when_user_org_different_to_family_org(
     assert db_family.family_category == "UNFCCC"
 
 
+def test_update_family_succeeds_when_user_org_different_to_family_org_super(
+    client: TestClient, data_db: Session, superuser_header_token
+):
+    setup_db(data_db)
+    new_family = create_family_write_dto(
+        title="apple",
+        summary="just a test",
+        geography="USA",
+        category=FamilyCategory.UNFCCC,
+        metadata={"color": ["pink"], "size": [0]},
+        collections=["C.0.0.3"],
+    )
+    response = client.put(
+        "/api/v1/families/A.0.0.1",
```

Comment:
This line jumped out to me. For my own understanding is it good practice to have `.` in an api call? 
Would it be better to add the collection in future to the json data or something?

Reply:
> "[Characters that are allowed in a URI but do not have a reserved purpose are called unreserved. These include uppercase and lowercase letters, decimal digits, hyphen, period, underscore, and tilde](https://www.rfc-editor.org/rfc/rfc3986)"

Full stops are allowed generally, but I think the situations to avoid are instances where a period or double period are sandwiched in between a part of the URI on their own e.g., `www.cpr.com/family/./` and `www.cpr.com/family/../` would redirect to `www.cpr.com/family/` and `www.cpr.com/` respectively.

> Would it be better to add the collection in future to the json data or something?

The collection "C.0.0.3" is already being added to the family "A.0.0.1" via JSON (line 299 in this case `json=new_family.model_dump()`) :)